ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/Praos/Common.hs

```diff
@@ -104,6 +104,12 @@ comparePraos tiebreakerFlavor =
     when' p comp a1 a2 =
         if p a1 a2 then comp a1 a2 else EQ
 
+    -- Only compare the issue numbers when the issuers and slots are identical.
+    -- Note that in this case, the VRFs must also coincide.
```

Comment:
Minor: I suggest saying instead that "Note that this case implies the VRFs also coincide".

Because, in this kind of context, "must" might have a connotation of something the user needs to be ensuring, a la the _contract_ of `comparePraos`.

Reply:
Adopted :+1: